constants/radio_constants.asm

```diff
@@ -123,3 +123,6 @@ NUM_PASSWORDS_PER_CATEGORY EQU 3
 	const BUENA_ITEM
 	const BUENA_MOVE
 	const BUENA_STRING
+
+; BuenaPrizeItems table length (see data/items/buena_prizes.asm)
+NUM_BUENA_PRIZES EQU 9
```

Comment:
`NUM_BUENA_PRIZES EQUS "(BuenaPrizeItemsEnd - BuenaPrizeItems) /  2"` and should go back into event/buena.asm

Reply:
Can't; the table is defined after the places where `NUM_BUENA_PRIZES` is used, and it won't compile. I can define `NUM_BUENA_PRIZES EQU 9` right after the table instead. (Previously there was no constant, just a hard-coded `db 9` and `db 1, 2, 3, 4, 5, 6, 7, 8, 9`.)

Reply:
The equs will still work, it doesn't have to be equ

Reply:
oh its rept

Reply:
I tried `EQUS` and that didn't work either. The expression's value is not defined at the time `rept` needs it.